configuration/tests/main.jsonnet

```diff
@@ -80,12 +80,12 @@ local metricsConfig = {
   writeEndpoint: 'http://%s.%s.svc.cluster.local:%d/api/metrics/v1/test/api/v1/receive' % [
     obs.api.service.metadata.name,
     obs.api.service.metadata.namespace,
-    obs.api.service.spec.ports[1].port,
+    obs.api.service.spec.ports[2].port,
```

Comment:
Seems like this was the wrong port previously?

Reply:
No I don't think so. But something has changed since these manifests were last generated causing the current port (`1`) to point at the internal  port on 8081. This change is required to avoid the diff in the generated manifests.